cypress/integration/init.spec.ts

```diff
@@ -2,16 +2,57 @@
 
 chai.should();
 describe('Cypress', () => {
-    it('is working', () => {
+
+    const mockAppState = {
+        smartContracts: ["greenContract@0.0.1", "blueContract@0.0.1"],
+        activeSmartContract: "greenContract@0.0.1"
+    }
+
+    beforeEach(() => {
         cy.visit('build/index.html').then((window) => {
-            window.postMessage('/transaction', '*'); // This is needed to trigger the `window.addEventListener`
+            // This is needed to trigger the `window.addEventListener`
+            window.postMessage('/transaction', '*'); 
```

Comment:
We should probably rename this file to indicate that these tests are for the 'transaction' component.